ext/civicrm_admin_ui/managed/SavedSearch_Manage_Mail_Accounts.mgd.php

```diff
@@ -2,6 +2,7 @@
 
 use CRM_CivicrmAdminUi_ExtensionUtil as E;
 
+// This SearchDisplay shows an editable-in-place field for Enabled? for all rows, including the bounce processing mail account, which cannot actually be disabled (you can change it to No, but it won't actually be disabled). So this is FIXME for when we can set rows to edit-in-place conditionally.
```

Comment:
@colemanw Any thoughts on how to handle this? I think it has come up in another context as well, but can't find that at the moment.

Reply:
Hmm, those edit-in-place links will check edit permissions before they render. So if the system had a rule like "you're not allowed to edit this" then it wouldn't be editable.
But lately we've been using [the Enable/Disable action links](https://github.com/civicrm/civicrm-core/pull/26842) instead of in-place-edit (or in-addition to, but I don't think both are strictly necessary). Those will have the same problem, so I guess we need to tweak the `checkAccess` callback for that entity to make sure it returns `false` for the ones that shouldn't be edited.